modules/android/nav-android.adoc

```diff
@@ -1,5 +1,4 @@
-:tag: -beta.1
-ifdef::releasetag[:tag: -{releasetag}]
+include::partial$_set_page_context_for_android.adoc[]
```

Comment:
You know, I wonder if this line is required?
Given that on the older version of line 48 below, {major} and {minor} etc. *were resolved*...
hmm, I guess those came from the Antora.yml, rather than the page context...

OK, never mind (but I'll try to remember to play with it at some point...)